core/templates/dev/head/pages/admin/admin.html

```diff
@@ -70,6 +70,20 @@
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <!-- About-page -->
+    <script src="/templates/dev/head/pages/about/about-page.module.js"></script>
+    <script src="/templates/dev/head/pages/about/about-page.controller.js"></script>
+
+    <!-- Donate page -->
+    <script src="/templates/dev/head/pages/donate/donate-page.module.js"></script>
+    <script src="/templates/dev/head/pages/donate/donate-page.controller.js"></script>
+
+    <!-- Creator Dashboard -->
+    <script src="/templates/dev/head/pages/creator_dashboard/creator-dashboard.module.js"></script>
+    <script src="/templates/dev/head/pages/creator_dashboard/creator-dashboard.controller.js"></script>
+
+    <!-- LADD1 {comment will be removed, used as achor atm -->
```

Comment:
Is this still required?

Reply:
Yeah for now it is. I will remove it when i am done with adding files ( this helps me replace this part with the script tags from terminal and add this part again)